api.py

```diff
@@ -37,24 +86,26 @@ def get(self):
 		selected_semester = self.request.get('semester')
 		
 		if selected_semester:
-			members = Member.query(Member.show == True, Member.semesters_paid == selected_semester).order(Member.name).fetch(limit=None)
+			# Avoid throwing error 500 if a bad semester string is supplied
```

Comment:
Very minor, but since the file needs edits anyway, would you mind terribly if I requested period at the end of the sentence?